mmv1/templates/terraform/operation.go.erb

```diff
@@ -69,8 +61,12 @@ func <%= product_name.camelize(:lower) -%>OperationWaitTimeWithResponse(config *
 <% end -%>
 
 func <%= product_name.camelize(:lower) -%>OperationWaitTime(config *Config, op map[string]interface{}, <% if has_project -%> project,<% end -%> activity, userAgent string, timeout time.Duration) error {
+  if val, ok := op["name"]; !ok || val == "" {
```

Comment:
I think there are two checks we want to make. First we can see if `done: true` is set and return the operation immediately if so. Next, we can check for a `name` and return `nil`- because we've probably misclassified an empty response (indicating a synchronous call) as an operation. If neither of those cases is true, we can wait on the operation.

I forget if MMv1 allows individual endpoints to have operations- I'd guess it does now, but didn't when we added https://github.com/GoogleCloudPlatform/magic-modules/pull/1396

Reply:
So, this PR does not represent a net change to OperationWaitTime - it just moves the check for "name" out of `create*Waiter`. 

I'm not clear on the behavior of Operations that use OperationWaitTime, so I am hesitant to make more changes here. I agree we could check if `done` is true and short-circuit (assuming that `done: True` is set on this kind of operation) but I don't know that there's much benefit, since it will already be checked as part of `OperationWait`

Reply:
SGTM- we don't need to add the done check. It'd be fine to do here if we wanted to, since it's already a dedicated change to this kind of behaviour (and only +- 16 LOC)